views/_includes/stepperMixin.pug

```diff
@@ -0,0 +1,3 @@
+mixin stepper(currentStep, totalSteps)
+  .stepper
+    p #{__('Screening questions')} (#{'Step '} #{__(currentStep)} #{' of '} #{__(totalSteps)})
```

Comment:
So we still need to translate `Step`, just not the numbers. IT'll end up looking like:
`#{__('Step ')} #{currentStep} #{__(' of ')} #{totalSteps}`

Reply:
I think this somehow made it back in when I rebased :s, I'll remove